go.sum

```diff
@@ -0,0 +1,136 @@
+github.com/bep/clocks v0.5.0 h1:hhvKVGLPQWRVsBP/UB7ErrHYIO42gINVbvqxvYTPVps=
```

Comment:
Review the necessity of each dependency, removing any that are no longer needed to reduce the overall size and complexity of the module's imports.